partner_contact_personal_information_page/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Personal information page for contacts",
+    "summary": "Add a page to contacts form to put personal information",
+    "version": "13.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "https://github.com/OCA/partner-contact",
+    "author": "Nicolas JEUDY,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "auto_install": False,
+    "depends": ["base"],
```

Comment:
Indent from the beginning, for easier extention. (for depends and data keys).

Reply:
Not possible.
Black will reformat this and Travis will fail.

Reply:
See: https://travis-ci.org/OCA/partner-contact/jobs/597517397#L310-L323

Reply:
OK, accepted